competitive-coding/Array/Soln.Question_2/Soln.Question_2/012_SORT.CPP

```diff
@@ -0,0 +1,26 @@
+#include<iostream.h>
+
+void main()
+{
+  int i,j,k,A[10]={0,2,0,1,1,2,1,0,1,2};
+  for(i=0;i<9;i++)
+  {
+     for(j=i+1;j<10;j++)
+     {
+       if(A[j]<A[i])
+       {
+	 k=A[j];
+	 A[j]=A[i];
+	 A[i]=k;
+       }
+       if(A[j]==A[i])
+       {
+	 i++;
+	 k=A[i];
+	 A[i]=A[j];
+	 A[j]=k;
+
+       }
+     }
+  }

```

Comment:
The time complexity is again O(n^2). As the question says, it needs to be done in O(n) time.
Don't use nested for loops.